system/scripts/npc/city/c_Klaipe.lua

```diff
@@ -1,216 +1,213 @@
---- [Item Merchant] Mirina --- 
+--- [Item Merchant] Mirina ---
 addnpc(20115, "ETC_20150717_013603", "c_Klaipe", 511, -1, -349, 45, "npc_itemmerchantmirina")
 function npc_itemmerchantmirina()
-	msg("Emilia_Select_1")
-	openshop("Klapeda_Misc")
+    msg("Emilia_Select_1")
```

Comment:
You should be using tabs, as per our coding conventions.

Reply:
Sorry...